app.js

```diff
@@ -98,7 +99,7 @@ function generateNewImages(){
   pictureArray[picturesOnScreenIndex[0]].numberOfTimesShowned++;
   pictureArray[picturesOnScreenIndex[1]].numberOfTimesShowned++;
   pictureArray[picturesOnScreenIndex[2]].numberOfTimesShowned++;
```

Comment:
try a loop here, this will reduce the amount of repeated code.

Additionally, you may way to add this to a function as it is used multiple places.